src/rpg_database/character_sheet/Race.java

```diff
@@ -0,0 +1,19 @@
+package rpg_database.character_sheet;
+
+import static rpg_database.character_sheet.common.CharacterSheetCommon.generateEnumText;
+
+public enum Race {

```

Comment:
Szerintem nyugodtan teszteld ebben a taskban és implementáld is hozzá a bázissebességeket. Még ha nem is használod őket semmire, akkor is ennek a tasknak a része

Reply:
ok